core/include/kernel/tee_ta_manager.h

```diff
@@ -93,6 +94,24 @@ struct tee_ta_session {
 	bool unlink;		/* True if session is to be unlinked */
 };
 
+#if defined(CFG_TA_STATS)
+#define MAX_DUMP_SESS_NUM	(16)
+struct tee_ta_dump_stats {
+	TEE_UUID uuid;
+	uint32_t panicked;	/* True if TA has panicked, written from asm */
+	uint32_t sess_num;  /* Number of opened session */
+	struct malloc_stats heap;
+};
+
+struct tee_ta_dump_ctx {
```

Comment:
This is only used in `core/kernel/tee_ta_manager.c` so please move it there.

Reply:
done.